git/release-branching.md

```diff
@@ -153,3 +153,5 @@ o---o---o master
                 J
                 o feature
 ```
+
+If you have more than one commit unique to the feature branch, you can't use `F^` in the `rebase --onto` command. Instead, you should use `F~3` or similar, depending on how many commits you have. If you don't know the number to use, you can still determine it (assuming you haven't pushed master to origin yet) with `git log --oneline origin/master..feature`, which will list the commits in the `feature` branch and not in the `origin/master` branch. (If you have pushed already, you will have to either examine the git log and rely on your memory, or else use the revlog. Good luck, and don't be shy about asking for help!)
```

Comment:
@kyptin I always use `<first-sha-of-branch>^` eg `fda232^` when doing rebase --onto.  I just use `git log --oneline` to determine what the first commit of the branch was.  Maybe the example needs to specify that `F` could be the first of many commits on a branch.